packages/common/src/store/reducers.ts

```diff
@@ -55,6 +55,8 @@ import trendingPlaylists from './pages/trending-playlists/slice'
 import trendingUnderground from './pages/trending-underground/slice'
 import trending from './pages/trending/reducer'
 import { TrendingPageState } from './pages/trending/types'
+import { PlaybackPositionState } from './playback-position'
+import playbackPosition from './playback-position/slice'
```

Comment:
Im not opposed to this as the slice name, just wondering what we might want to call this? im thinking a little into the future, how we it might be more that just "position", it could be playback rate, playback timestamp, number, status, maybe play-count or something? just wanted to flag the idea on what would best encapsulate that.... maybe just "PlaybackState"?

Reply:
funny enough i called it playback state first and then changed it when I had to make the PlaybackStateState type lol